src/i18n/locales/en-US.json

```diff
@@ -498,6 +498,7 @@
   "cmps.terminal.OpenTerminalButton.menu": "Launch Terminal",
   "cmps.terminal.OpenTerminalButton.info": "Run '{{cmd}}' commands directly on the node",
   "cmps.terminal.OpenTerminalButton.openMsg": "Launching the terminal...",
+  "cmps.appExit.DockerContainerShutdown.shutdownMsg": "Shutting down...",
```

Comment:
Change the key to `cmps.common.DockerContainerShutdown` and move this line to the up with the other `cmps.common.*` keys.